app/package.json

```diff
@@ -109,7 +109,6 @@
     "core-js": "^3.19.1",
     "cypress": "8.5.0",
     "cypress-plugin-stripe-elements": "^1.0.2",
-    "dom-testing-library": "^2.3.2",
```

Comment:
just checking if these two removed libraries are all good to go as they're not mentioned in the pr desc

Reply:
These are old package versions. The new library packages are installed already. 
https://www.npmjs.com/package/dom-testing-library
https://www.npmjs.com/package/react-testing-library

Reply:
They moved everything to the `@testing-library` org on npm.